Update CDM and analysis

sroyos opened the pull request:
- Update CDM version to 0.1.0 version

- Add analysis scripts (report (.qmd) and auxiliar scripts (.R))

sroyos committed:
Update CDM and analysis

- Update CDM version to 0.1.0 version

- Add analysis scripts (report (.qmd) and auxiliar scripts (.R))

cienciadedatosysalud commented on CITATION.cff:
Cambia repository-code: "https://github.com/user/repository"

cienciadedatosysalud commented on zenodo.json:
El fichero tiene que ser un fichero oculto que se denomine " .zenodo.json"

sroyos committed:
Requested changes

sroyos committed:
Update cdm version and analysis

Updated CDM version:

- Added co-authors

Updated analysis:

- Added script: 'all_ind_calc_perfil.R'
- Updated scripts to improve output files
